mlir/lib/Dialect/MIOpen/Generator/Conv2dGenerator.cpp

```diff
@@ -175,11 +175,11 @@ LogicalResult Conv2dGenerator::parseConvConfig(const char *arguments) {
 
```

Comment:
I'd prefer to add a dimension size == 5 check before you entered the layout translation function. That function wasn't designed to handle a layout size larger than 5.

Reply:
Done